src/main/proto/Report.proto

```diff
@@ -65,24 +65,58 @@ message Schedule {
 message ReportLog {
     string task_status = 1;
     string task_executed = 2;
-    string threshold_met = 3;
```

Comment:
change it to camel notiation

Reply:
I think the official naming convention suggests to use snake case https://developers.google.com/protocol-buffers/docs/style?csw=1